test/utils/navigator.py

```diff
@@ -69,12 +68,11 @@ def send_and_navigate(send: Callable[[], RESPONSE], navigate: Callable[[], None]
 class TezosUseCaseAddressConfirmation(UseCaseAddressConfirmation):
     """Extension of UseCaseAddressConfirmation for our app."""
 
-    # Y-285 = common space shared by buttons under a key displayed on 2 or 3 lines
-    SHOW_QR = (CENTER.x, 285)
+    tappable_center = TappableCenter
```

Comment:
SHOW_QR is used at line 340. Need to change there too.

Reply:
Done